client/mocks/builders_test.go

```diff
@@ -88,6 +90,30 @@ func buildEcsClusterMock(t *testing.T, ctrl *gomock.Controller) client.Services
 	return services
 }
 
+func buildCloudfrontDistributionsMock(t *testing.T, ctrl *gomock.Controller) client.Services {
+	m := mocks.NewMockCloudfrontClient(ctrl)
+	services := client.Services{
+		Cloudfront: m,
+	}
+
+	ds := cloudfrontTypes.DistributionSummary{}
+	err := faker.FakeData(&ds)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
you can do this in one line i.e if err := faker.FakeData(&ds); err != nil { return t.Fatal(err) }